Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -93,11 +106,22 @@ public final class ManagePledgeViewModel:
     let backing = graphBackingEnvelope
       .map { $0.backing }
 
-    self.endRefreshing = graphBackingEvent
+    let endRefreshingWhenProjectFailed = fetchProjectEvent
```

Comment:
Why not `fetchProjectEvent.errors().ignoreValues()`?

Reply:
Sure yeah that should work 👍